users-api/routes/signup.js

```diff
@@ -20,6 +25,18 @@ router.post('/', (req, res) => {
   }
   else
   {
+    getDb(db, => {
+      db.collection('users')
+      .insert({
+        user: req.body.username,
+        pwd: req.body.password
+      }, (err, result) => {
+        if (err) {
+          return res.status(500).json(err);
+        }
```

Comment:
What happens if it doesn't error? We should move the successful response to here